component/__manifest__.py

```diff
@@ -5,13 +5,18 @@
     "name": "Components",
     "summary": "Add capabilities to register and use decoupled components,"
     " as an alternative to model classes",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.0",
     "author": "Camptocamp," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/connector",
     "license": "LGPL-3",
     "category": "Generic Modules",
     "depends": ["base"],
-    "external_dependencies": {"python": ["cachetools"]},
+    "external_dependencies": {
+        "python": [
+            "cachetools",
+            "mock",
```

Comment:
why do we need mock here?

Reply:
Because it's used on tests. Do you think that it's not necessary because the tests are not the code itself? Just in the same sense as @guewen mentioned in https://github.com/akretion/connector/pull/2#discussion_r509111937?

Reply:
Ok I see, to be installed and executed in our projects this module doesn't need mock.
Only to be tested it needs it. Probably it'll be tested only on OCA and on developer computer. 
That's what @simahawk mean I suppose.

Reply:
It's a good point

Reply:
in any case mock is a built-in lib since py3... you won't need it anyway https://docs.python.org/3.5/library/unittest.mock.html

Reply:
Thanks for the info

Reply:
Wasn't this line to remove then?

Reply:
yep, unattended

Reply:
Finally no more mock in this PR @guewen @simahawk

Reply:
Thanks!